### ISSUE TITLE
VisInteractions RuntimeError

### ISSUE THREAD

**Issue by nicholascwho** [asrhou/NATMI#39]
Hi there, thank you for this easy to use package. I have successfully extracted the edges, but am having some problems with the last step. The VisInteractions function seems to keep facing RuntimeErrors. I tried to reduce the number of edges kept but it doesn't seem to work. The code manages to create the new folder so I believe it is a problem with the plotting? Thank you.

**Comment by asrhou** [comment]
Hi, could you show me the error messages? Thanks

**Comment by M-Arafat** [comment]
yeah, it creates empty folder with no plots inside AND there is no error messages!

**Comment by asrhou** [comment]
If there are no error messages, maybe there are no edges left. Could you try to lower the thresholds?
